src/PufferfishAligner.cpp

```diff
@@ -634,6 +638,8 @@ void processReadsSingle(single_parser *parser,
     // aconf.allowOverhangSoftclip = mopts->allowOverhangSoftclip;
     // aconf.allowSoftclip = mopts->allowSoftclip;
     aconf.computeCIGAR = (mopts->computeCIGAR and !mopts->noOutput);
+    aconf.endBonus = mopts->endBonus;
```

Comment:
Again, we might need to have `endBonus` in the aconf struct anymore.

Reply:
I have removed it from here as well.